docs/source/usage/plugins/countParticles.rst

```diff
@@ -33,7 +33,7 @@ Output
 In the output file ``e_macroParticlesCount.dat``, there are three columns.
 The first is the integer number of the time step.
 The second is the number of macro particles as integer - useful for exact counts.
-And the third is the number of macro particles in scintific floating point notation - provides better human readability.
+And the third is the number of macro particles in scientific floating point notation - provides better human readability.
```

Comment:
I believe `floating-point` used as adjective should be hyphenated (probably present in a lot of places).

Reply:
Yep, there is the term `floating point`, e.g. in `decimal floating point` but the adjective `floating-point number`, `floating-point arithmetic`, `floating-point format`, etc.

Not critical, can be changed partially here or consequently for all occurrences in a follow-up PR.

Reply:
Okay - should be changed with a follow-up pull request.